app/src/androidTest/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivityTest.java

```diff
@@ -87,7 +88,7 @@
     public void signIn() {
         CountDownLatch latch = new CountDownLatch(1);
 
-        fAuth = FirebaseAuth.getInstance();
+        fAuth = AuthenticationInstanceProvider.getAuthenticationInstance();
```

Comment:
You make an abstraction for the sign in part ?

Reply:
No, that's not an abstraction! This is simply a call to a method created by Kepler to instantiate the Firebase Authentication service in a cleaner way!